src/cache/cache_las.c

```diff
@@ -686,6 +686,7 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
         }
 
         upd = prev_upd = NULL;
+        squashed = false;
```

Comment:
I'd suggest moving the initialization of `squashed` and `prev_upd` to the loop where they're used, but I wouldn't argue for it.